exercises/storeWithObjects.js

```diff
@@ -48,9 +30,12 @@ function newSofa(name, price) {
  * and `false` otherwise.
  */
 function userCanAffordSofa(user, sofa) {
-  // This is your job. :)
-
-  return user['budget'] >= sofa['price'];
+  if (user['budget'] > sofa['price']) {
```

Comment:
You want `>=` here.

And like above, just `return user['budget'] >= sofa['price'];`